test/test_fr_little_endian.js

```diff
@@ -22,6 +22,26 @@ test("Test - Single expression", function() {
     }
 
 
+    var text = "1er Août 2012";
```

Comment:
there shuld be test for edge case like `20er Août 2012` and `0er Août 2012` ect.